connexion/operation.py

```diff
@@ -131,7 +131,7 @@ def _request_end_lifecycle_decorator(self):
         return EndOfRequestLifecycleDecorator(self.api, self.get_mimetype())
 
 
-class Operation(SecureOperation):
+class Swagger2Operation(SecureOperation):
```

Comment:
Can you add a `Operation = Swagger2Operation` in the end of the module in case someone is importing it?